index.md

```diff
@@ -148,13 +156,21 @@ Technical
 * DNS-OARC 29: Estimating impact of the (E)DNS flag day [abstract](https://indico.dns-oarc.net/event/29/contributions/644/), [slides](https://indico.dns-oarc.net/event/29/contributions/644/attachments/632/1018/edns.pdf), [video](https://youtu.be/rneu1lnJmUI?list=PLCAxS3rufJ1cOBd1D4K4QJFmLcSypixh3&t=3001)
 * DNS-OARC 28: First announcement [abstract](https://indico.dns-oarc.net/event/28/contributions/515/), [slides](https://indico.dns-oarc.net/event/28/contributions/515/attachments/490/799/Removing_EDNS_Workarounds.pdf), [video](https://www.youtube.com/watch?v=9YYH8JFH_bY&feature=youtu.be&t=5198)
 
+
+<a name="tools"></a>
+
```

Comment:
I don't expect these are needed.  We automatically have these targets on current dnsflagday.net :
```html
<h1 id="tools">Tools</h1>
```
and I have a vague feeling that using `id` on headers is better than `<a>` tags.  (I know very little about present-day HTML; I'd have used unpaired version if choosing anchors `<a ... />`.)